### ISSUE TITLE
Create experiment from scratch from the UI

### ISSUE THREAD

**Issue by c-abate** [activeshadow/minimega#83]
Right now to create an experiment from a new topology and scenario it looks like you have to run a command from the terminal first to load the topologies and scenarios into the data store.  It would be really beneficial if it was possible to load these from the phenix UI or have phenix do it automatically if they exist in a certain directory on the server.  Reason being that I've noticed on a couple projects that usability is important to adoption and not everyone is comfortable using the terminal to get things done. The more features we can built into the phenix UI so that we don't need to use the terminal would be beneficial.

**Comment by activeshadow** [comment]
@c-abate yeah I've been wanting to add some code that will load any configs present in a particular directory, like /etc/phenix/configs` or `/phenix/autoload` or something like that.

**Comment by aherna** [comment]
@activeshadow the new minimega release 2.7 lets you draw a topology and export to v5 json of phenix. Working on updating the schema to support v6. It also allows the user to run the drawn topology directly

**Comment by activeshadow** [comment]
@aherna supporting v6 only requires that you wrap your existing v5 schema with some header details.

```
{
  "apiVersion": "phenix.sandia.gov/v0",
  "kind": "Topology",
  "metadata": {
    "name": "<name you want to use for topology>",
  },
  "spec": {
    <v5 JSON definition here>
  }
}
```
